### ISSUE TITLE
Background glitches out when weather changes

### ISSUE THREAD

**Issue by ghost** [multimokia/MAS-Submod-Auto-Atmos-Change#35]
Not sure quite how to describe this, so I've attached a poor video example of what I mean.  

When the weather changes, the spaceroom background disappears and you can only see Monika and the desk and the calendar.  When this happens you can kind of see the weather background cycle through a few images before locking onto the correct one, then the spaceroom background reappears.  This happens regardless of which location she is in.

It's been happening for months, but previously it was working fine so I'm not sure how to account for it.  I've been trying to capture it so you can see what I mean.  I managed to catch it during a recent storm, please excuse 'Labyrinth' in the foreground - I didn't expect the weather to change so suddenly.

https://user-images.githubusercontent.com/61102477/186205547-fb1d5604-1efe-49ae-94c4-f3b0d84b3343.mp4

.

**Comment by multimokia** [comment]
I'm working on a total overhaul of the submod, hopefully this might be addressed in it.
Do note if you're pulling from source, the build won't run on a normal MAS install. It only works if you're running MAS on renpy 8.

**Comment by ghost** [comment]
I don't believe I'm pulling from source - the submod was installed manually via the release zip back in the day and has been updated by the submod updater plugin since then.

Thank you, I'm looking forward to seeing the new and improved Auto Atmos Change!
